cypress/integration/granules_spec.js

```diff
@@ -38,88 +38,93 @@ describe('Dashboard Granules Page', () => {
       // shows a list of granules
       cy.getFakeApiFixture('granules').as('granulesListFixture');
 
+      const numGranulesToTest = 4;
```

Comment:
This was just an optimization, this test is really slow, and repeated for every granule in the fixture, this just tests the first 4 instead of all 10.  but takes about 90secs off of the test time.